src/components/pages/Categories/Detail.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div>
+    <category-detail
+      :category-id="categoryId"
+      :done-message="doneMessage"
+      :error-message="errorMessage"
+      :access="access"
+      :category="category"
+      :loading="loading"
+      :disabled="loading ? true: false"
+      @clear-message="clearMessage"
+      @handle-submit="handleSubmit"
+      @edited-category="editedCategory"
+    />
+  </div>
+</template>
+
+<script>
+import CategoryDetail from '@Components/molecules/CategoryDetail/index.vue';
+
+export default {
+  components: {
+    categoryDetail: CategoryDetail,
+  },
+  computed: {
+    access() {
+      return this.$store.getters['auth/access'];
+    },
+    selectCategoryName() {
+      const selectCategoryName = this.$store.state.categories.targetCategory.name;
+      return selectCategoryName;
+    },
+    categoryId() {
+      const { id } = this.$route.params;
+      return id;
+    },
+    category() {
+      const category = this.$store.state.categories.targetCategory.name;
+      return category;
+    },
```

Comment:
[imo]
そのままreturnしても良いかなと思いました！

```suggestion
    category() {
      return this.$store.state.categories.targetCategory.name;
    },
```